### ISSUE TITLE
OMI is only read/write for OMIDevs group

### ISSUE THREAD

**Issue by jeffaco** [Microsoft/omi#5]
Hi folks,

I needed to change how OMI is accessed due to additional team members; I needed to change the granularity of access.

We have an additional team now:
1. @MSFTOSSMgmt/omsdevs : Controls access to all repositories except OMI
2. @MSFTOSSMgmt/omidevs : Controls access to the OMI repository

If you are doing active development in OMI, then you need to be in the OMIDevs team. If you only need read access to to OMI project, then just being in OMSDevs should be sufficient (which everyone getting this is except for JohnA and Andy).

Bottom line: This shouldn't affect anyone except the few of us directly contributing to the OMI project.

If you have any problems, let me know, and I'll take care of it.

Note: I would anticipate that, as time progresses, more granular teams will be set up. In all cases, the OMSDevs team will retain access to the repositories that they need, although some may become read-only.

/Jeff
